distribution/tscript.js

```diff
@@ -7812,7 +7812,8 @@ module.createCanvasIcon = function(description)
 	
 	// zoom
 	// TODO: programmaticly detect whenever zoom changes
-	let zoom = window.devicePixelRatio;
+	let zoom = 2; // Good enough
```

Comment:
Why wasn't `window.devicePixelRatio` good enough? Didn't you want to be flexible with scaling?

Reply:
see other conversation